vernac/declareDef.mli

```diff
@@ -14,8 +14,9 @@ open Decl_kinds
 val get_locality : Id.t -> kind:string -> Decl_kinds.locality -> bool
 
 val declare_definition : Id.t -> definition_kind ->
```

Comment:
Style remark: I prefer putting optional arguments before all the remaining ones, in the style of Coq implicit arguments. By no means a request for change.